content/introduction/why-dolt/performance-characteristics.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Performance Characteristics
```

Comment:
This is not Performance. I would call the Scalability, Availaibility, and Performance or something.